buildSrc/src/main/resources/japicmp_exceptions.json

```diff
@@ -1,3 +1,6 @@
 {
   "Class org.apache.geode.management.builder.GeodeClusterManagementServiceBuilder": "Moved internal class to fix split packages between geode-core and geode-management",
+  "Class org.apache.geode.management.api.ClusterManagementOperation": "Fixed missing @Experimental annotation",
+  "Method org.apache.geode.management.api.ClusterManagementOperation.getEndpoint()": "Fixed missing @Experimental annotation",
```

Comment:
aren't the methods implicitly experimental, and therefore ignored now, by the class being marked Experimental?  So these last 2 lines might be unnecessary?